docs/nais-application/migration.md

```diff
@@ -0,0 +1,192 @@
+# Migrate from GHCR to GAR
+
+## Introduction
+
+This document describes how to migrate from GHCR (GitHub Container Registry) to GAR (Google Artifact Registry).
+
+## Why migrate?
+
+GHCR serves as a container registry hosted by GitHub, but it has certain limitations, such as imposing strict rate
```

Comment:
container registry or "OCI registry", Maybe it's long term beneficial to point out that you can put any oci blobs in there.